src/components/Header/Header.js

```diff
@@ -48,22 +71,41 @@ export default class Header extends Component {
                   {this.renderNavItem('/recipes', 'Recipes')}
                   {this.renderNavItem('/calculator', 'Calculator')}
                   {this.renderNavItem('/flavors', 'Flavors')}
-                  <NavDropdown title="User">
-                    {this.renderNavDropdownItem('/user/profile', 'Profile')}
-                    {this.renderNavDropdownItem('/user/recipes', 'Recipes')}
-                    {this.renderNavDropdownItem('/user/favorites', 'Favorites')}
-                    {this.renderNavDropdownItem(
-                      '/user/flavor-stash',
-                      'Flavor Stash'
-                    )}
-                    {this.renderNavDropdownItem(
-                      '/user/shopping-list',
-                      'Shopping List'
-                    )}
-                    {this.renderNavDropdownItem('/user/settings', 'Settings')}
-                  </NavDropdown>
-                  {this.renderNavItem('/login', 'Login')}
-                  {this.renderNavItem('/register', 'Register')}
+                  {loggedIn ? (
+                    <NavDropdown title="User">
+                      {this.renderNavDropdownItem('/user/profile', 'Profile')}
+                      {this.renderNavDropdownItem('/user/recipes', 'Recipes')}
+                      {this.renderNavDropdownItem(
+                        '/user/favorites',
+                        'Favorites'
+                      )}
+                      {this.renderNavDropdownItem(
+                        '/user/flavor-stash',
+                        'Flavor Stash'
+                      )}
+                      {this.renderNavDropdownItem(
+                        '/user/shopping-list',
+                        'Shopping List'
+                      )}
+                      {this.renderNavDropdownItem('/user/settings', 'Settings')}
+
+                      {this.renderNavDropdownItem('/dashboard', 'Dashboard')}
+                      <NavDropdown.Item
+                        as={NavLink}
+                        to="#"
+                        className="nav--link-custom"
+                        activeClassName="active"
+                        onClick={this.logoutUser}
+                      >
+                        Logout
+                        <br />
+                      </NavDropdown.Item>
+                    </NavDropdown>
+                  ) : (
+                    ''
+                  )}
+                  {!loggedIn ? this.renderNavItem('/login', 'Login') : ''}
```

Comment:
Might want to wrap these two in a `<Fragment>` so you only need one ternary.

Reply:
This has been implemented